plugin/execute_command.py

```diff
@@ -0,0 +1,36 @@
+from .core.registry import client_for_view, LspTextCommand
+from .core.settings import client_configs
+from .core.protocol import Request
+
+try:
+    from typing import List, Optional, Dict, Any
+    assert List and Optional and Dict and Any
+except ImportError:
+    pass
+
+
+class LspExecuteCommand(LspTextCommand):
+    def __init__(self, view):
+        super().__init__(view)
+
+    def run(self, lsp_command) -> None:
+        self._commands = []   # type: List[str]
+        for config in client_configs.all:
+            if config.commands:
+                self._commands.extend(config.commands)
+
+        if len(self._commands) > 0:
+            self.view.window().show_quick_panel(self._commands, self._on_done)
+
+    def _handle_response(self, response: 'Optional[Any]') -> None:
```

Comment:
@rwols given the response can be `Any` what do you think should be the way to format and display the response?
Should we update the diagnostic panel ?

Reply:
What does metals return? Have you tried printing it to the status bar (`sublime.status_message`) and/or the console (`print`)?

Reply:
response is empty so far. I will display a popop in case of an error and probably just log the response for now